packages/editor/src/client/atoms/listInput.tsx

```diff
@@ -64,7 +64,7 @@ const ListItem = SortableElement(
           <DragHandle disabled={itemDisabled} />
         </div>
         <Panel bodyFill style={{width: '100%'}}>
-          <div style={{minHeight: '100%'}}>
+          <div style={{minHeight: '100%', display: 'flex'}}>
```

Comment:
I guess this is needed so that the error message shows up properly?

Reply:
Yes indeed